### DISCUSSION TITLE
Linux

### DISCUSSION THREAD

**Discussion by nicks-boson** [carson-katri/dream-textures#108] [Q&A]
What a great plug-in is this. Congrats! In the read me I see a description that’s applied to windows. But is this also working under Linux?

**Comment by NathanSalapat** [comment]
I've gotten it working in Linux Mint 20.2 with a GTX 1060.

**Comment by Anyeos** [comment]
https://github.com/carson-katri/dream-textures/discussions/231#discussioncomment-3900798

**Comment by crazyrobinhood** [comment]
It's working like charm
